src/components/RefinementList/__tests__/RefinementList-test.js

```diff
@@ -56,8 +56,8 @@ describe('RefinementList', () => {
         </div>
       </div>
     );
-    expect(out.props.children[0].key).toEqual('facet1');
-    expect(out.props.children[1].key).toEqual('facet2');
+    expect(out.props.children[0][0].key).toEqual('facet1');
```

Comment:
I fail at understanding why this test changed, the underlying DOM tree is different than before I guess but looking at the changes of RefinementList I do not see it. Maybe because of changes inside Template?